<?xml version="1.0"?>

<launch>
  
  <include file="$(find ugv_course_launch)/launch/roundbot_nav_sim.launch" />  
  
  <node pkg="state_space_example" type="state_space_example" name="odom_node" >
    <remap from="twist" to="roundbot/twist" />
  </node>

  <node pkg="nav_stack_example" type="nav_stack_node" name="nav_stack_node" output="screen" >
    <rosparam file="$(find nav_stack_example)/yaml/costmap_params.yaml" ns="global_costmap" />
    <rosparam file="$(find nav_stack_example)/yaml/global_planner_params.yaml" ns="global_planner" /> 
    <rosparam file="$(find nav_stack_example)/yaml/local_planner_params.yaml" ns="local_planner" />
    <rosparam file="$(find nav_stack_example)/yaml/local_costmap_params.yaml" ns="local_costmap" />
  </node>

  <node pkg="tf" type="static_transform_publisher" name="world_to_map" args="0 0 0 0 0 0 /world /map 30" />

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find nav_stack_example)/rviz/rviz_config.rviz" respawn="true" />

</launch>
